src/rules/flake8_bandit/rules/try_except_pass.rs

```diff
@@ -0,0 +1,45 @@
+use ruff_macros::derive_message_formats;
+use rustpython_ast::{Expr, ExprKind, Located, Stmt, StmtKind};
+
+use crate::ast::types::Range;
+use crate::checkers::ast::Checker;
+use crate::define_violation;
+use crate::registry::Diagnostic;
+use crate::violation::Violation;
+
+define_violation!(
+    pub struct TryExceptPass;
+);
+impl Violation for TryExceptPass {
+    #[derive_message_formats]
+    fn message(&self) -> String {
+        format!("Try, Except, Pass detected.")
```

Comment:
I know this is exactly what Bandit uses, but our conventions would probably be more like:

```
format!("`try`-`except`-`pass` detected")
```

Or commas in lieu of dashes. But wrapping except keyword in backticks, and removing the period.

(Sorry for the nit!)

Reply:
Thanks - fixed 👍